great-idea-website/index.html

```diff
@@ -32,9 +32,9 @@
 
     <section class="head">
       <div class="quote">
-         <h1>Innoaavation
+         <h1>Innoavation
```

Comment:
This is what a comment would look like. I know we had gone over this last night as well.